var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -77,10 +97,12 @@ class Vtk(CMakePackage):
 
     depends_on('ffmpeg', when='+ffmpeg')
 
+    depends_on('eigen@2.91:', when='@8.2:')
+    depends_on('double-conversion', when='@8.2:')
     depends_on('expat')
     depends_on('freetype')
     depends_on('glew')
-    depends_on('hdf5')
+    depends_on('hdf5+hl')  # netcdf needs highlevel-API
```

Comment:
This doesn't really belong here.  See the discussion over on #11306

Reply:
This should be:
```
depends_on('hdf5')
```
That will mesh well with NetCDF.  And it's correct, since this package doesn't DIRECTLY need `+hl`.